Gemfile

```diff
@@ -12,5 +12,5 @@ group :development, :test do
   gem "guard-rubocop", require: false
   gem "rubocop", "~> 0.63.1", require: false
   gem "rubocop-rspec", require: false
-
+  gem "byebug", require: true
```

Comment:
Can you move this to gemspec file, please

Reply:
I think for a reason of good practices, it was better to have gems in the Gemfile but I am not 100% sure, @mwolman  could you give us more information?

Also, yeah, maybe it's better to remove 2.7 since it's support finished.

What do you mean by "check the bundler version"?

Reply:
Also, rubocop is being checked already

Reply:
@adlamas If this is a gem project, I prefer manage all gems in the `.gemspec` file instead `Gemfile`